planetiler-basemap/src/main/java/com/onthegomap/planetiler/basemap/generated/OpenMapTilesSchema.java

```diff
@@ -165,9 +165,11 @@ final class FieldValues {
     /** Complex mappings to generate attribute values from OSM element tags in the water layer. */
     final class FieldMappings {
       public static final MultiExpression<String> Class =
-        MultiExpression.of(List.of(MultiExpression.entry("dock", matchAny("waterway", "dock")),
+        MultiExpression.of(List.of(
+          MultiExpression.entry("dock", matchAny("waterway", "dock")),
           MultiExpression.entry("river", or(matchAny("water", "river"), matchAny("waterway", "riverbank"))),
-          MultiExpression.entry("lake", matchAny("waterway", "")), MultiExpression.entry("ocean", FALSE)));
+          MultiExpression.entry("lake", matchAny("natural", "water"))
+        ));
```

Comment:
This changes the logic to match OMT's logic - waterway=dock is a `dock`, river areas are `river` and all other water areas are water.  The previous waterway='' matchAny didn't make much sense and gave inexplicable errors I wasn't able to track down.  This version matches OMTs logic and passes all water layer test cases.

Reply:
If the order of expressions in the list is non-deterministic, then we'll need to change this logic.